src/config/options.rs

```diff
@@ -183,6 +183,9 @@ configuration_option_enum! { WriteMode:
     Checkstyle,
     // Output the changed lines (for internal value only)
     Modified,
+    // Displays how much of the input was processed, but if anything was modified, rustfmt quits
+    // with exit code 0. This option is intended to be run as part of a CI pipeline.
```

Comment:
I think the exit code is backwards - it should quit with `1` if anything was changed. It displays a diff, if there is one, not progress exactly.

Reply:
D'oh—you're right. I'll fix in subsequent commits.

Reply:
Fixed/resolved.